examples/check_examples

```diff
@@ -47,6 +47,50 @@ set -e
 export PSYCLONE=${PWD}/../bin/psyclone
 export KSTUBGEN=${PWD}/../bin/genkernelstub
 
+# Ensure we use the configuration file distributed with PSyclone and
+# not a (potentially modified) local version.
+export PSYCLONE_CONFIG=${PWD}/../config/psyclone.cfg
+
+# ----------------------------------------------------------------- #
+# Handle any command-line options. Currently we only support one.
+# This code taken from https://stackoverflow.com/questions/192249/
+# how-do-i-parse-command-line-arguments-in-bash
+COMPILE=0
```

Comment:
My 2 cents: I would actually not agree. I think a shell script is just the natural language for this, with python we might have to use subshells (e.g. to call ``make`` or so).

Reply:
I agree with @hiker here - we'd have to shell-out for the majority of the tasks performed in the script. We could possibly switch to a Makefile as I think that's a more natural fit with what's being done. It would then automatically give us support for e.g. running just the LFRic examples.

Reply:
Agreed. Issue #713 has been created.